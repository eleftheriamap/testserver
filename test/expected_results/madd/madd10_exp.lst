
test/exp/test/gen/madd/madd10_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d154eb9c 	sub	x28, x28, #0x53a, lsl #12
   4:	d16743de 	sub	x30, x30, #0x9d0, lsl #12
   8:	11260f9c 	add	w28, w28, #0x983
   c:	9b1e739a 	madd	x26, x28, x30, x28
  10:	8a000000 	and	x0, x0, x0
