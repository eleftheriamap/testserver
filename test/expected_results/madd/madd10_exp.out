Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = ff6333444a556983
X27    = 0000000000000000
X28    = 00000000ffac6983
X29    = 0000000000000000
X30    = ffffffffff630000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd154eb9c
0x00000004 : 0xd16743de
0x00000008 : 0x11260f9c
0x0000000c : 0x9b1e739a
0x00000010 : 0x8a000000
